kbase-extension/static/kbase/gallery/README.md

```diff
@@ -0,0 +1,44 @@
+# Demos
+
+These demos are static html pages which demonstrate some relatively simple and isolatable aspect of the Narrative. This simple demo system is particularly suitable for ui components.
```

Comment:
This looks a whole lot like Storybook. So why not just use Storybook?

Reply:
Just wanted something simple to be able to preview components w/out having to fire up the narrative and without having to introduce something like storybook to the codebase. I'll just remove this. It is useful for me, and I thought it might be for others as well.

Reply:
I'm definitely not against something like this for the Narrative, this does seem a bit out of scope for this PR, though, and could use some discussion.

Thanks for removing for now.

Reply:
No problem. It was the only way I could actually use the tab control with all combinations of options, since they  don't actually all exist in the codebase; so it served its purpose for me at least :)